examples/guestbook/frontend-controller.yaml

```diff
@@ -1,12 +1,12 @@
-apiVersion: v1
```

Comment:
I thought about this, and then I think that deployment is also a class of "pod controller" so I decided to keep the name. WDYT?